db/migrate/20241102232405_add_index_to_pokemons.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexToPokemons < ActiveRecord::Migration[7.1]
+  def change
+    add_index :pokemons, :name
+    add_index :pokemons, :id
+  end
+end
```

Comment:
I imagine this helps structure the database by giving each pokemon a unique attribute, but how does this differ from the purpose the primary key provides?